### ISSUE TITLE
How to run the code 

### ISSUE THREAD

**Issue by abhigoku10** [prabhuomkar/pytorch-cpp#26]
@prabhuomkar thanks for the this codebase , just wanted to knwo run your tutorials ??

**Comment by mfl28** [comment]
If you followed the steps in "Getting Started" in the readme, the binaries should be in 
```
build/tutorials/<tutorial_category>/<tutorial_name>/
``` 
So for example, the executable for the logistic regression tutorial should be in 
```
build/tutorials/basics/logistic_regression/
```

**Comment by abhigoku10** [comment]
@mfl28 @prabhuomkar  i followed the same steps but not able to see any binary files 
![image](https://user-images.githubusercontent.com/22094053/72055631-30798980-32f1-11ea-9b52-eef6c41b5881.png)
in the mentioned location i have only the above contents . i am performing this activity on cpu , i have disable cuda==None

**Comment by mfl28** [comment]
Can you tell me exactly the cmake commands you used?
Did you execute `cmake --build build` from the pytorch-cpp directory after generating the build system?

**Comment by abhigoku10** [comment]
@mfl28 Thanks for instant response .Please find the commands i have done 
git clone https://github.com/prabhuomkar/pytorch-cpp.git
cd pytorch-cpp
cmake -B build -D CUDA_V=none -D DOWNLOAD_DATASETS=OFF -D CMAKE_PREFIX_PATH= ./
cmake -build build

**Comment by mfl28** [comment]
Thanks!
> cmake -build build

You need to use **two** hyphens before the `build` parameter, like this:
`cmake --build build`

Some additional info for the first cmake command (`cmake -B build ...`):
* It isn't necessary to specify `-D CUDA_V=none` as it is set by default.
* You don't have to set  `-D DOWNLOAD_DATASETS=OFF` if the data was already previously downloaded as the cmake script checks if files are already present (in `pytorch-cpp/data`) and if so, doesn't redownload them. If, on the other hand, you really don't want to download the datasets at all through the cmake script, you'll have to do it manually, otherwise most of the tutorials won't run.
* Also you don't need to specify `-D CMAKE_PREFIX_PATH=./`, as by default the cmake script will look for a previously downloaded libtorch in `pytorch-cpp/libtorch` if `CMAKE_PREFIX_PATH` is not specified.

So, in short, if you want to build on Linux, using libtorch for CPU (and not using an existing local version outside of `pytorch-cpp/libtorch`) it should be enough to do:
```
cmake -B build
cmake --build build
```
@prabhuomkar We might want to clarify some of these things better in the readme.

**Comment by abhigoku10** [comment]
@mfl28  thanks for the steps now i am able to build the binaries

**Comment by prabhuomkar** [comment]
> We might want to clarify some of these things better in the readme.
@mfl28  Agreed, I will send a PR for this.

**Comment by prabhuomkar** [comment]
@abhigoku10 let's close this if there are not more queries or issues in running the code?
